community/examples/gke-tpu-v4/gke-tpu-v4.yaml

```diff
@@ -0,0 +1,141 @@
+# Copyright 2025 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+blueprint_name: gke-tpu-v4
+
+vars:
+  # The following variables should be over-written in the deployment.yaml file.
+  # Your GCP Project ID
+  project_id:
+
+  # This should be unique across all of your Cluster
+  # Toolkit Deployments.
+  deployment_name: gke-tpu-v4
+
+  # The GCP Region used for this deployment.
+  region:
+
+  # The GCP Zone used for this deployment.
+  zone:
+
+  # The number of TPU slices to create
+  num_slices:
+
+  # The TPU placement topology for pod slice node pool.
+  tpu_topology:
+
+  # The number of nodes to be created in each nodepool
+  static_node_count:
+
+  # Cidr block containing the IP of the machine calling terraform.
+  # To allow all (IAM restrictions still enforced), use 0.0.0.0/0
+  # To allow only your IP address, use <YOUR-IP-ADDRESS>/32
+  authorized_cidr:
+
+  # The name of the compute engine reservation of A4-High nodes in the form of
+  # <project>/<reservation-name>/reservationBlocks/<reservation-block-name>
+  extended_reservation:
+
+
+deployment_groups:
+- group: primary
+  modules:
+  - id: gke-tpu-v4-net
+    source: modules/network/vpc
+    settings:
+      network_name: $(vars.deployment_name)-net-0
```

Comment:
if it doesn't add much, I would remove the network name and mtu, perhaps even subnetwork name